src/opds_dumper.cpp

```diff
@@ -165,7 +165,8 @@ std::map<std::string, std::string> iso639_3 = {
   {"rmr", "Caló"},
   {"rmy", "romani shib"},
   {"roa", "romance languages"},
-  {"twi", "twi"}
+  {"twi", "twi"},
+  {"hbs", "srpskohrvatski"},
```

Comment:
This list appears to be sorted for convenience. I don't mind if it's not sorted but a general good habit is to respect conventions.

Reply:
Eagle eye :)

Kept the comma from mgautierfr's comment.